specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2024-01-01-preview/workspaceRP.json

```diff
@@ -980,19 +966,22 @@
           }
         },
         "x-ms-examples": {
-          "List Workspace Keys": {
-            "$ref": "./examples/Notebook/listKeys.json"
+          "List Endpoint": {
+            "$ref": "./examples/Endpoint/list.json"
           }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
         }
       }
     },
-    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{workspaceName}/listStorageAccountKeys": {
-      "post": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{workspaceName}/endpoints/{endpointName}": {
```

Comment:
>"/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{workspaceName}/endpoints/{endpointName}": { [](http://example.com/codeflow?start=4&length=167)

Why no DELETE method?

Reply:
This API is still work in progress, currently we only onboard two type of endpoint which we consider as built-in type and we delete along with the workspace and not allow user to delete them, we will add DELETE method in next API version when we onboard more supported type